config/locales/client.en.yml

```
@@ -4586,6 +4586,9 @@ en:
               other: "%{count} users have the new email domains and will be added to the group."
             automatic_membership_associated_groups: "Users who are members of a group on a service listed here will be automatically added to this group when they log in with the service."
             primary_group: "Automatically set as primary group"
+          alert:
+            primary_group: "Since this is a primary group for its members, the name '%{group_name}' will be used in CSS classes which can be viewable by anyone. Please ensure the name is safe."
+            flair_group: "Since this group has flair for its members, the name '%{group_name}' will be used in CSS classes which can be viewable by anyone. Please ensure the name is safe."
```

```suggestion
            flair_group: "Since this group has flair for its members, the name '%{group_name}' will be visible to anyone."
```

I removed "Please ensure the name is safe" in both strings because it can be ambiguous, "safe" can refer to the safety of the string itself and it could confuse users. The main goal of the notice anyway is to let admins know that the group name is public, and the shorter string does that.